Docs: Review fixes of developer guide intro pages (#1401)
advancedtelematic/aktualizr: docs/ota-client-guide/modules/ROOT/pages/workflow-overview.adoc

== Installing updates on devices

In OTA Connect, we generally assume that updates will be OSTree images or complete firmware images to flash onto secondary ECUs, but in fact OTA Connect can be used to send all kinds of other updates.footnote:[For more information on using OTA Connect for other types of update, please contact us at link:mailto:otaconnect.support@here.com[otaconnect.support@here.com].] For now, let's look at the default OSTree update process.

.OTA Connect device update flow
****
. The client polls OTA Connect servers periodically to check if there are any new updates.
. If there are, the client receives a metadata file directing it to install a particular image.
. The metadata is cryptographically checked for validity following the https://uptane.github.io[Uptane] https://uptane.github.io/uptane-standard/uptane-standard.html[specification].

> . If all the checks pass, the client downloads the image. In the case of an OSTree update, the commit representing the image is downloaded from TreeHub, only actually requiring the download of missing objects and diffs.

Reviewer: ```suggestion
. If all the checks pass, the client downloads the image. In the case of an OSTree update, the commit representing the image is downloaded from TreeHub, which only actually requires downloading any objects missing on the client.
```
